pyclam/manifold.py

```diff
@@ -1181,15 +1174,56 @@ def clear_cache(self):
     def depth(self) -> int:
         return len(self.layers) - 1
 
+    def get_dot_file_string(self, graph_name):
+        """
+        checks over every cluster in the manifold, finding their parent edges and all edges between two clusters on the
+        same level, and then writes the information for each cluster and each edge onto a string that can be written
+        on a dot file
+        """
+        # starting the graph string with the graph name (an inputted value), and the root cluster
+        dot_file_string = 'graph ' + str(graph_name) + '{ \n' +\
+                          f'	edge[style=solid, penwidth="5",labeldistance="10"]\n' \
+                          f'	{self.root}[label="NODE: {self.root.name}\\nLFD: ' \
+                          f'{round(self.root.local_fractal_dimension, 3)}", ' \
+                          f'color="{self.root.color}", style=filled]\n'
```

Comment:
When using f-strings, you don't need to use + for concatenation  nor do you need \ for line breaks